pages/glossary/layout-table.md

```diff
@@ -0,0 +1,14 @@
+---	
+title: Layout table	
+key: layout-table	
+---	
+
+Table that have no tabular mark-up are interpreted by assistive technologies as layout table. According to WCAG this includes, but is not limited to:	
+
+- No header [`cells`](https://www.w3.org/TR/html52/tabular-data.html#cell), that is [`th`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th) elements, or [`td`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td) elements with ARIA roles [`rowheader`](https://www.w3.org/TR/wai-aria-1.1/#rowheader) and [`columnheader`](https://www.w3.org/TR/wai-aria-1.1/#columnheader).	
+- No [`caption`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption) element.	
+- No (or empty) `summary` attribute.	
+- No `scope` or `headers` attribute on table cells.	
+- Table with single column or row, or table with less than two rows containing two valid data cells are automatically marked as layout table by assistive technologies. A cell is considered valid:	
```

Comment:
Is it less "that two rows" or "two or less rows"? Because the first is equal to table with one row